text/0000-std-aware-cargo.md

```diff
@@ -0,0 +1,289 @@
+- Feature Name: cargo_the_std_awakens
+- Start Date: 2018-02-09
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Currently, the `core` and `std` components of Rust are handled in a different way than Cargo handles other crate dependencies. This causes issues for non-mainstream targets, such as WASM, Embedded, and new not-yet-tier-1 targets. The following RFC proposes a roadmap to address these concerns in a consistent and incremental process.
+
```

Comment:
You might want to give a brief summary of the "how it is achieved" here as well.

Reply:
I was a little bit worried about repeating myself as this is explained without detail (in the Guide Level), and explained with detail (in the Reference Level).

I am unsure how to summarize without repeating the content of the Guide Level Explanation verbatim. I am open to suggestions!

Reply:
I'll try to think of something to suggest.. :)